.ado/jobs/desktop.yml

```diff
@@ -155,9 +139,6 @@ jobs:
                   Write-Host "##vso[task.setvariable variable=Desktop.IntegrationTests.Filter]$newValue"
                 displayName: Update Desktop.IntegrationTests.Filter to exclude RNTester integration tests
 
-            - ${{ if eq(matrix.UseFabric, true) }}:
-              - template: ../templates/enable-fabric-experimental-feature.yml
```

Comment:
I think you can delete enable-fabric-experimental-feature.yml too?

Reply:
No, we use it for Universal